tools/mtouch/Application.cs

```diff
@@ -2175,7 +2175,13 @@ public void BundleAssemblies ()
 					return false;
 				if (PackageManagedDebugSymbols)
 					return false;
-				if (IsInterpreted (Assembly.GetIdentity (path)))
+				/* FIXME: should be `if (IsInterpreted (Assembly.GetIdentity (path)))`.
```

Comment:
was an issue filled for the FIXME ? if not then ... :)
if/when so please add a link to it